### DISCUSSION TITLE
Unable to render parallax and parallax layer

### DISCUSSION THREAD

**Discussion by AdonsFCO** [pmndrs/react-spring#1881] [Support]
I installed the library called spring/parallax in this way:

 yarn add @react-spring/parallax@9.4.5-beta.1

All this because I count use the other version on react 18. The problem here is that not matter what I do, the layers are not rendering. 

````
import react from "react";
import { Parallax, ParallaxLayer } from "@react-spring/parallax";

function App() {
  return (
    <Parallax>  
      <ParallaxLayer>
        <h1>moto</h1>
      </ParallaxLayer>
      <ParallaxLayer>
        <h1>hello moto</h1>
      </ParallaxLayer>
    </Parallax>
  );
}

export default App;

```

I tried a lot of things like using vite, using create-create-react-app because I thought that was a problem related with the package, and i'm still thinking that is that. (I'm new on react)

**Comment by kindoflew** [comment]
Hello!

A couple ideas:
* Did you also install `@react-spring/web`?
* Did you add the `pages` prop?

If those don't help, could you reproduce the issue using something like codesandbox? thanks!
